src/handlers/admin_handler.py

```diff
@@ -88,27 +73,27 @@ def start(self, admin_token:str, _ip):
             fields=["start_time"]
         )
 
-    def set_extra_time(self, admin_token:str, extra_time:int, _ip, token:str=None):
+    @Validators.validate_admin_only
+    def set_extra_time(self, extra_time:int, token:str=None):
         """
         POST /admin/set_extra_time
         """
-        self._validate_token(admin_token, _ip)
-
         if token is None:
             Database.set_meta("extra_time", extra_time)
+        elif Database.get_user(token) is None:
```

Comment:
Should we use a validator for this too? (by passing a boolean parameter "can_be_none" to the validator, for example)